Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -206,10 +206,10 @@
 
 - type: entity
   parent: BaseTraitorRuleNoRandomObjectives
-  id: SpyVsSpy5TC
+  id: SpyVsSpy3TC
```

Comment:
Just rename this to something like `SpyVsSpyLowTC` so the name doesn't have to change with the value